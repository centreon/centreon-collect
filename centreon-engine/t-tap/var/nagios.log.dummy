This is a dummy nagios.log file. Will be archived
