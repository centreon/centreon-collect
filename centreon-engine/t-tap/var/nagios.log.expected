[1242949698] LOG ROTATION: HOURLY

[1242949698] LOG VERSION: 2.0

